modules/get-started/pages/quick-start.adoc

```diff
@@ -6,7 +6,7 @@
 
 {description}
 
-This quickstart introduces you to Redpanda Self-Managed in a self-hosted environment with a minimal setup. Docker Compose provides a straightforward and reproducible setup process, whether you're exploring Redpanda for the first time or testing new configurations. The option between a single broker and a three-broker setup helps you test Redpanda for simplicity and resilience.
+This quickstart introduces you to Redpanda Self-Managed in a self-managed environment with a minimal setup. Docker Compose provides a straightforward and reproducible setup process, whether you're exploring Redpanda for the first time or testing new configurations. The option between a single broker and a three-broker setup helps you test Redpanda for simplicity and resilience.
```

Comment:
This reads strangely. Please stick with the original.

Reply:
reverted.